acceptance_tests/features/pages/reporting_unit.py

```diff
@@ -56,12 +56,36 @@ def get_associated_respondents():
     return respondents
 
 
+def get_associated_respondents_with_pending_email():
+    respondents_table = browser.find_by_name('tbl-respondents-for-survey')
+    rows = respondents_table.find_by_tag('tbody').find_by_tag('tr')
+    respondents = [
+        {
+            "enrolmentStatus": row.find_by_id('enrolment-status').value,
+            "name": row.find_by_name('tbl-respondent-details').first.find_by_name('tbl-respondent-name').value,
+            "email": row.find_by_name('tbl-respondent-details').first.find_by_name('tbl-respondent-email').value,
+            "pending_email": row.find_by_name('tbl-respondent-details')
+            .first.find_by_name('tbl-respondent-pending-email').value,
```

Comment:
This line fails when there are any respondents without a pending email address present